x-pack/plugins/ml/public/application/jobs/jobs_list/components/delete_job_modal/delete_job_modal.tsx

```diff
@@ -0,0 +1,162 @@
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiSpacer,
+  EuiModal,
+  EuiOverlayMask,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiButtonEmpty,
+  EuiButton,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+
+import { deleteJobs } from '../utils';
+import { DELETING_JOBS_REFRESH_INTERVAL_MS } from '../../../../../../common/constants/jobs_list';
+import { DeleteJobCheckModal } from '../../../../components/delete_job_check_modal';
+
+type ShowFunc = (jobs: Array<{ id: string }>) => void;
+
+interface Props {
+  setShowFunction(showFunc: ShowFunc): void;
+  unsetShowFunction(): void;
+  refreshJobs(): void;
+}
+
+export const DeleteJobModal: FC<Props> = ({ setShowFunction, unsetShowFunction, refreshJobs }) => {
+  const [deleting, setDeleting] = useState(false);
+  const [modalVisible, setModalVisible] = useState(false);
+  const [jobIds, setJobIds] = useState<string[]>([]);
+  const [canDelete, setCanDelete] = useState(false);
+
+  useEffect(() => {
+    if (typeof setShowFunction === 'function') {
+      setShowFunction(showModal);
+    }
+    return () => {
+      if (typeof unsetShowFunction === 'function') {
+        unsetShowFunction();
+      }
+    };
+  }, []);
+
+  function showModal(jobs: any[]) {
+    setJobIds(jobs.map(({ id }) => id));
+    setModalVisible(true);
+    setDeleting(false);
+  }
+
+  function closeModal() {
+    setModalVisible(false);
+    setCanDelete(false);
+  }
+
+  function deleteJob() {
+    setDeleting(true);
+    deleteJobs(jobIds.map((id) => ({ id })));
+
+    setTimeout(() => {
+      closeModal();
+      refreshJobs();
+    }, DELETING_JOBS_REFRESH_INTERVAL_MS);
+  }
+
+  if (modalVisible === false || jobIds.length === 0) {
+    return null;
+  }
+
+  if (canDelete) {
+    return (
+      <EuiOverlayMask>
+        <EuiModal data-test-subj="mlDeleteJobConfirmModal" onClose={closeModal}>
+          <EuiModalHeader>
+            <EuiModalHeaderTitle>
+              <FormattedMessage
+                id="xpack.ml.jobsList.deleteJobModal.deleteJobsTitle"
+                defaultMessage="Delete {jobsCount, plural, one {{jobId}} other {# jobs}}?"
+                values={{
+                  jobsCount: jobIds.length,
+                  jobId: jobIds[0],
+                }}
+              />
+            </EuiModalHeaderTitle>
+          </EuiModalHeader>
+          <EuiModalBody>
+            <p>
+              {deleting === true ? (
+                <div>
+                  <FormattedMessage
+                    id="xpack.ml.jobsList.deleteJobModal.deletingJobsStatusLabel"
+                    defaultMessage="Deleting jobs"
+                  />
+                  <EuiSpacer />
+                  <div style={{ textAlign: 'center' }}>
+                    <EuiLoadingSpinner size="l" />
+                  </div>
+                </div>
+              ) : (
+                <>
+                  <FormattedMessage
```

Comment:
The line spacing in this message doesn't look quite right to me. Can it be increased?

![image](https://user-images.githubusercontent.com/7405507/102117976-f7b62e80-3e36-11eb-8bd5-6bff1f096ff1.png)